### ISSUE TITLE
Issue with wepack build (Error: custom keyword definition is invalid: data/errors should be boolean)

### ISSUE THREAD

**Issue by NicolasCARPi** [epoberezkin/ajv-errors#26]
Hello,

I had this error with my webpack build:

~~~log
/srv/http/elabftw/node_modules/webpack-cli/bin/cli.js:231
                                throw err;
                                ^

Error: custom keyword definition is invalid: data/errors should be boolean
    at Ajv.addKeyword (/srv/http/elabftw/node_modules/ajv/lib/keyword.js:65:13)
    at module.exports (/srv/http/elabftw/node_modules/ajv-errors/index.js:10:7)
    at Object.<anonymous> (/srv/http/elabftw/node_modules/schema-utils/src/validateOptions.js:22:1)
    at Module._compile (/srv/http/elabftw/node_modules/v8-compile-cache/v8-compile-cache.js:178:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:745:10)
    at Module.load (internal/modules/cjs/loader.js:626:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:566:12)
    at Function.Module._load (internal/modules/cjs/loader.js:558:3)
    at Module.require (internal/modules/cjs/loader.js:663:17)
    at require (/srv/http/elabftw/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/srv/http/elabftw/node_modules/schema-utils/src/index.js:7:25)
    at Module._compile (/srv/http/elabftw/node_modules/v8-compile-cache/v8-compile-cache.js:178:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:745:10)
    at Module.load (internal/modules/cjs/loader.js:626:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:566:12)
    at Function.Module._load (internal/modules/cjs/loader.js:558:3)
    at Module.require (internal/modules/cjs/loader.js:663:17)
    at require (/srv/http/elabftw/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/srv/http/elabftw/node_modules/webpack/lib/SourceMapDevToolPlugin.js:13:25)
    at Module._compile (/srv/http/elabftw/node_modules/v8-compile-cache/v8-compile-cache.js:178:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:745:10)
    at Module.load (internal/modules/cjs/loader.js:626:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:566:12)
    at Function.Module._load (internal/modules/cjs/loader.js:558:3)
    at Module.require (internal/modules/cjs/loader.js:663:17)
    at require (/srv/http/elabftw/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/srv/http/elabftw/node_modules/webpack/lib/WebpackOptionsApply.js:16:32)
    at Module._compile (/srv/http/elabftw/node_modules/v8-compile-cache/v8-compile-cache.js:178:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:745:10)
    at Module.load (internal/modules/cjs/loader.js:626:32)
error Command failed with exit code 1.
~~~

**Changing [line 14 of index.js](https://github.com/epoberezkin/ajv-errors/blob/master/index.js#L14) and setting it `errors` to `true` fixed the issue**. But I'm not sure of the consequences, I just know it lets me build my files :)

Do you think a fix could be published for this issue? I'm using 1.0.1 of ajv-errors and latest versions of wepack and friends.

**Comment by DBosley** [comment]
This issue popped up for me today too.

**Comment by DarkLink363** [comment]
To fix this error, just change:
Line 14: `errors: 'full',`
to:
Line 14: `errors: true`
in file **node_modules\ajv-errors\index.js**

**Comment by josiahdahl** [comment]
You can resolve it by installing `ajv@6.8.1` https://github.com/epoberezkin/ajv/issues/941

Here's the commit that broke it, in release 6.9.0 https://github.com/epoberezkin/ajv/commit/47c8fc90ad5737d2243de1b9986783f5fe51b703, file `lib/keyword.js:13`

**Comment by epoberezkin** [comment]
'full' is correct value for errors, the validation was wrong. Fixed in ajv 6.9.1
